Tests/PackageDescription4Tests/JSONTests.swift

```diff
@@ -1,30 +0,0 @@
-/*
- This source file is part of the Swift.org open source project
-
- Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
- Licensed under Apache License v2.0 with Runtime Library Exception
-
- See http://swift.org/LICENSE.txt for license information
- See http://swift.org/CONTRIBUTORS.txt for Swift project authors
-*/
-
-import XCTest
-@testable import PackageDescription4
-
-class JSONTests: XCTestCase {
```

Comment:
We will need to remove the corresponding entry in XCTestManifest, right?

Reply:
Done.